### DISCUSSION TITLE
Math mode centering block is unintuitive

### DISCUSSION THREAD

**Discussion by toihr** [typst/typst#1288] [Ideas]
I think that the way to pre and append a space to a math string to tell it to display in its own block is unintuitive. It is also not easy visualy discernable in larger formulas I belive. 
I would propose to make it such that when one enters a formula in a new line with no text that it then creates a new block.

**Comment by Enivex** [comment]
You don't need to use a space, line break also works

```
$
a^2 + b^2 +c^2
$
```
If you want to be super explicit you can even do
```
#math.equation(block: true,$
a^2+b^2
$)
```

**Reply by laurmaedje** [reply]
@Enivex I think your second example is missing dollar signs.

**Reply by Enivex** [reply]
> @Enivex I think your second example is missing dollar signs.

I had a brain fart

**Reply by toihr** [reply]
Ah ok I have not considered this. Would it be possible to add that to the documentation as that seems to not be part of it. But this does indeed solve my problems I had with it. 
Thank you.
